client/src/app/users/create-user/create-user.component.ts

```diff
@@ -73,27 +73,15 @@ export class CreateUserComponent implements OnInit {
         return this.createForm.controls;
     }
 
-    checkWhitespace() {
-        
-        if (!this.controls.firstName.value.trim()) {
-            this.controls.firstName.setErrors({'required': true});
-        } else {
-            this.controls.firstName.setErrors({'required': false});
-        }
-
-        if (!this.controls.lastName.value.trim()) {
-            this.controls.lastName.setErrors({'required': true});
-        } else {
-            this.controls.lastName.setErrors({'required': false});
-        }
+    trimOnBlur() {
```

Comment:
- Maybe you can introduce the control whose value shall be trimmed as a parameter of the method.
- I am not sure about naming. `OnBlur` part of the name is telling when it is done, not what is done.